templates/_composition/_shared/Page.Camera.AddResource/Controls/CameraControl.xaml.cs

```diff
@@ -94,98 +184,59 @@ public bool IsInitialized
                     photoOrientation = PhotoOrientation.FlipHorizontal;
                 }
 
-                return await ReencodeAndSavePhotoAsync(stream, photoOrientation);
+                var photo = await ReencodeAndSavePhotoAsync(stream, photoOrientation);
+                PhotoTaken?.Invoke(this, new CameraControlEventArgs(photo));
+                return photo;
```

Comment:
Set _capturing to false

Reply:
@CarteKiwi You're right, it's already fixed.